miio/huizuo.py

```diff
@@ -87,6 +231,16 @@ def __init__(
         lazy_discover: bool = True,
         model: str = MODEL_HUIZUO_PIS123,
     ) -> None:
+
+        if model in MODELS_WITH_FAN_WY:
```

Comment:
You could implement a new device class per device group. Each device class will pop-up as new parameter/device at the miiocli.

Reply:
Split with classes